### PR TITLE
Community contributions documentation

### PR DESCRIPTION
**Pull request by quadrophobiac**
I'm moving the draft text from ODI Contributor Guide: 2nd Draft into the README of this repo.
I'd like some feedback on how this reads - is there anything I've missed and should include? @JeniT @Floppy @ldodds @davetaz @pikesley @pezholio

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+ 
+The Toolbox grew out of ODI Labs engagement with the open data software ecosystem. Noticing gaps in the types of data tools available Labs developed MVP experimental products intended to plug those gaps. We hope these provocations get everyone thinking. We’d like for the community working in open data software to either fill the identified gaps in other ways, use our work to continue filling those gaps (even locate as yet unidentified gaps!) or contribute to our tools to improve their utility. 
+ 
+We look to OSS success stories like Ruby, Github and Adapt and we want to build an ecosystem of OSS tools that support Open Data - like OKFN’s JKAN. We know we can’t do this all by ourselves, nor would we see this as the best way forward - we aspire to build with developments in the ecosystem. We want to build an OSS community around the Toolbox and hope you can be part of it!
```
Comment (Stephen-Gates) [inline README.md]: Do you mean Open Knowledge International's CKAN project?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+The Roadmap tracks future directions for each of the toolbox. You can consider it as the feature backlog for the entire toolbox and per toolbox application. 
+You can also vote and comment on cards to express which ones are most important to you
+
+### Contribute code to our Toolbox
+
+see the next sections for full details
+
+## Contributing Code
+
+If you’re a developer interested in Open Data familiar with any of the following technologies we’d like to interest you in collaborating on our Toolbox
```
Comment (Stephen-Gates) [inline README.md]: If you’re a developer interested in Open Data **and** familiar with any of the following technologies **,** we’d like to interest you in collaborating on our Toolbox

**Comment by quadrophobiac** [comment]
Thanks for the feedback on this - I have parcelled out the contents of this readme to a wiki within this repository in case you have any further suggestions for improvement https://github.com/theodi/toolbox/wiki

**Comment by quadrophobiac** [comment]
Expanding scope of this PR to include image assets for use in Wiki

**Comment by quadrophobiac** [comment]
additions made, not showing up here

**Pull request closed without merging**